### ISSUE TITLE
Way to ignore some pink blocks

### ISSUE THREAD

**Issue by SunSerega** [maruohon/litematica#21]
When i was watching 19+20-th scichaft witch farms stream - someone asked about what is pink on top of world eater (it was snow).

And so i got an idea - can you add way to add some blocks to ignore list, for them to not show pink overlay, like snow, would probably help finding mistakes in world eater.

Also it would work good with water. Not having your whole screen being pink would really help when you build under water.

And in my personal experience - there are specific blocks in every situation, that just really not helping by being pink.

Can you add this kind of setting? Preferably as list, where you can easily add and remove items. And maybe also way to temporally turn off the whole list, without deleting all items.

**Comment by maruohon** [comment]
Yes that kind of a system is planned. Basically a way to define lists of valid blockstate or block substitutions or ignored blocks/states, depending on the use case. Such lists would be usable for the verifier, material list, overlay rendering and possibly also to not render certain types of blocks at all. The implementation details and how the lists will be managed and selected/used etc. hasn't yet been thought out.
